src/Compilers/CSharp/Test/Semantic/Semantics/LookupTests.cs

```diff
@@ -2108,6 +2108,22 @@ static void Main(string[] args)
             Assert.NotNull(symbolInfo.Symbol);
         }
 
+        // TODO: it's not clear that this test is meaningful.
```

Comment:
TODO?

Reply:
Yeah, test plan suggests including a LookupSymbols test, but I'm genuinely not sure what would be meaningful here.￼ @333fred in case you have any suggestions.

Reply:
Have a test with a constructor that takes a parameter of type T, and make sure that you can get constructor information (I can't remember whether you need to use `GetSymbolInfo` or `LookupSymbols` for that.

Reply:
For `LookupSymbols` we'd want a test like `LookupConstrAndDestr` with `Attr1</*bind*/string/*bind*/>` to confirm we can tell the IDE symbols are in scope there.